client-sdk/ts-web/rt/src/types.ts

```diff
@@ -18,6 +18,43 @@ export interface CoreCallDataPublicKeyQueryResponse {
     public_key: KeyManagerSignedPublicKey;
 }
 
+/**
+ * Response to the RuntimeInfo query.
+ */
+export interface CoreRuntimeInfoQueryResponse {
+    runtime_version: runtime_info.Version;
+    state_version: number;
+    modules: { [key: string]: runtime_info.ModuleInfo };
+}
+
+export namespace runtime_info {
+    export interface Version {
```

Comment:
I noticed that for the current version 0.1.0, this gets deserialized into `{minor: 1}` in TS -- the zeros are omitted. That's intentional when encoding / on the wire (`#[cbor(skip_serializing_if = "num_traits::Zero::is_zero")]`), but looks a little funny in TS; we'd need the equivalent of `#[cbor(default)]`. I guess we're OK with it for now? We don't have a deserialization framework in place, AFAICT, where we could implant default values.

Reply:
yeah, we're living with it.